Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/PlaylistScript.js

```diff
@@ -278,6 +290,58 @@ window.__firefox__.includeOnce("Playlist", function($) {
           }
           return document_createElement.call(this, tag);
       };*/
+      
+      // Needed for Japanese videos like tver.jp which literally never loads automatically
+      Object.defineProperty(window.__firefox__, 'playlistProcessDocumentLoad', {
+        enumerable: false,
+        configurable: false,
+        writable: false,
+        value:
+        function() {
+          onReady(function() {
+            let videos = getAllVideoElements();
+            let audios = getAllAudioElements();
+            
+            if (videos.length == 0 && audios.length == 0) {
+              $(function() {
+                $.postNativeMessage('$<message_handler>', {
+                  "securityToken": SECURITY_TOKEN,
+                  "state": document.readyState
+                });
+              })();
+              return;
+            }
+            
+            videos.forEach(function(node) {
+              observeNode(node);
+              notifyNode(node, 'video', true, true);
+            });
+
+            audios.forEach(function(node) {
+              observeNode(node);
+              notifyNode(node, 'audio', true, true);
+            });
+          });
+          
+          // Timeinterval is needed for DailyMotion as their DOM is bad
+          var interval = setInterval(function() {
+            getAllVideoElements().forEach(function(node) {
+              observeNode(node);
+              notifyNode(node, 'video', true, true);
+            });
+
+            getAllAudioElements().forEach(function(node) {
+              observeNode(node);
+              notifyNode(node, 'audio', true, true);
+            });
+          }, 1000);
+
+          var timeout = setTimeout(function() {
+            clearInterval(interval);
+            clearTimeout(timeout);
```

Comment:
This doesn't seem needed? Timeout's are only executed once, so clearTimeout here would do nothing

Reply:
Correct. Fixed. I just have a habit of cleaning up variables manually :D